go.mod

```diff
@@ -54,4 +54,6 @@ require (
 // Workaround for github.com/containerd/containerd issue #3031
 replace github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
 
+replace github.com/golang/protobuf => github.com/golang/protobuf v1.3.1
```

Comment:
Can you explain "why" as like the replace directive above?

Reply:
gogo/protobuf is not compatible with APIv2 golang/protobuf 1.4, the same issue is described in [contaierd/ttrpc#62](https://github.com/containerd/ttrpc/issues/62). Also if we ever use go mod tidy, it gets automatically upgrade to v1.4 so we need to manually downgrade it w/ replace.

Reply:
Thanks. @alakesh Can you add that on go.mod file? People could find Royce's comment from `git annotate` though.